pkgs/development/tools/misc/ycmd/default.nix

```diff
@@ -7,12 +7,12 @@
 
 stdenv.mkDerivation rec {
   name = "ycmd-${version}";
-  version = "2017-11-05";
+  version = "2018-06-14";
 
   src = fetchgit {
-    url = "git://github.com/Valloric/ycmd.git";
-    rev = "0ca16e7ba3a9078b697befdce64104e61829aa3b";
-    sha256 = "0bs94iv521ac2n53n3k8mw3s6v0hi3hhxhjsr0ips3n99al8wndi";
+    url = "https://github.com/Valloric/ycmd.git";
```

Comment:
Thank you for swapping to HTTPS!  A lot of corporate networks and firewalls block SSH and FTP traffic from third party sites, but let HTTPS through; I was going to submit a PR to swap the url for `ycmd` but as always the wonderful Nix community is one step ahead :+1: